guides/common/modules/proc_copying-a-content-view.adoc

```diff
@@ -0,0 +1,30 @@
+[id="Copying_a_Content_View_{context}"]
+= Copying a content view
+
+You can copy a content view in the {ProjectWebUI} or you can use the Hammer CLI to copy an existing content view into a new content view.
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-copying-a-content-view_{context}[].
+
+[NOTE]
+====
+A copied content view does not have the same history as the original content view.
+Version 1 of the copied content view begins at the last version of the original content view.
+As a result, you cannot promote an older version of a content view from the copied content view.
+====
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Lifecycle* > *Content Views*.
+. Select the content view you want to copy.
+. Click the vertical ellipsis icon and click *Copy*.
+. In the *Name* field, enter a name for the new content view and click *Copy content view*.
+
+[id="cli-copying-a-content-view_{context}"]
+.CLI procedure
+* Copy the content view by using Hammer:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# hammer content-view copy --name _My_original_CV_name_ \
+--new-name _My_new_CV_name_
+----
+.Verification
+* The copied content view appears on the *Content views* page.
```

Comment:
The thing is that in CLI, there might be a list command to verify that the copied CV appears in its output. This way, the user wouldn't have to leave the CLI for the verification.
What do you think about having two Verifications - one below the Procedure for WebUI and second below the CLI Procedure for Hammer.

Reply:
The same verification?

Reply:
No.

Reply:
I'm checking with a SME to verify.